AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -211,6 +211,8 @@ mail.ru##.viqeo-embed
 severpost.ru#?##content div.col:has(> div[style] a[href="https://Instagram.com/severpost.ru"])
 ! https://github.com/AdguardTeam/AdguardFilters/issues/57975
 blog.calltouch.ru##.mainsub > div#blog-subscribe + div.mainsub__rightside
+! https://github.com/AdguardTeam/AdguardFilters/issues/74116
+iqiyi.com##iqpdiv[data-cupid=container],div[adid]
```

Comment:
Hi.
We prefer to not add more than one selector to one rule. Please change to two rules